wiki/Manual.md

```diff
@@ -3733,15 +3733,29 @@ if ($exchange->has['createMarketOrder']) {
 
 #### Market Buys
 
-In general, when placing a `market buy` or `market sell` order the user has to specify just the amount of the base currency to buy or sell. However, with some exchanges market buy orders implement a different approach to calculating the value of the order.
+When placing a **market buy** or **market sell** order the user usually has to specify just the amount of the **quote** currency to buy or sell.
 
-Suppose you're trading BTC/USD and the current market price for BTC is over 9000 USD. For a market buy or market sell you could specify an `amount` of 2 BTC and that would result in _plus or minus_ 18000 USD (more or less ;)) on your account, depending on the side of the order.
+- **Ex.** For a trade using symbol `BTC/USD` with a current BTC market price of 9000 USD, to place a market buy/sell you would specify an `amount` of 2 BTC, resulting in ± 18000 USD on your account (+18000 for a market sell of 2 BTC, -18000 for a market buy of 2 BTC)
 
-**With market buys some exchanges require the total cost of the order in the quote currency!** The logic behind it is simple, instead of taking the amount of base currency to buy or sell some exchanges operate with _"how much quote currency you want to spend on buying in total"_.
+Some exchanges require the total cost of market buy orders in the **base** currency
```

Comment:
Are you sure you're not confusing the `base` and `quote` terms?

the cost is the `quote` currency